theano/tensor/tests/test_basic.py

```diff
@@ -5174,7 +5169,7 @@ def test_flatten_outdim2():
     f = inplace_func([a], c)
     assert numpy.all(f(a_val) == a_val)
 
-    utt.verify_grad(Flatten(2), [a_val])
+    utt.verify_grad(flatten, [a_val])
```

Comment:
Do you mean replacing this line with:

    from functools import partial
    flatten_2 = partial(flatten, outdim=2)
    utt.verify_grad(flatten_2, [a_val])
?